sdk/search/Build-SearchPackages.ps1

```diff
@@ -0,0 +1,9 @@
+$scriptDir = Split-Path $MyInvocation.MyCommand.Path -Parent
+$packagesDir = "$scriptDir\..\..\binaries\packages"
+
+if (!(Test-Path -Path $packagesDir)) {
+    mkdir $packagesDir
+}
+
+msbuild "$scriptDir\..\..\build.proj" /t:CreateNugetPackage /p:Scope="sdk\search\Microsoft.Azure.Management.Search"
+msbuild "$scriptDir\..\..\build.proj" /t:CreateNugetPackage /p:Scope="sdk\search"
```

Comment:
So scope `sdk\search` is shorthand for the data plane? Is that a common pattern for other Azure services as well? It seems a bit counter-intuitive.

Reply:
For better or worse the management and data-plane toolsets are a little different, however if you are scoping to just search you can use create one solution that has all the projects and call `dotnet pack` on that solution to produce the packages for these libraries.

Reply:
Since there are various data-plane packages I kept the solution file in the service directory for all data-plane. Then a separate one for the management plane which is in its package directory.

Reply:
What I'm trying to figure out is if we're doing something fundamentally different than other service teams, and if not, if this change reflects the same practice that others are using.

Reply:
So far most service teams don't actively contribute to both their management and data-plane libraries at the same time so usually there isn't a solution that includes them both. However for you guys I have no problem with you have a solution that references them both and it will help with the scenario you are trying to accomplish with this script.

Reply:
We actually don't actively develop the data plane and management plane SDKs together (they are owned by separate teams); we just have these cross-project dependencies.

It's interesting... I would have assumed that every data plane's tests needed the management plane to provision the resources to be tested, so it surprises me that this isn't more common.

Reply:
The majority of service teams are generally using static test resources as opposed to provisioning as part of the test steps, which is why most don't do that.